js/acquia_lift.debugger.js

```diff
@@ -0,0 +1,1885 @@
+/**
+ * Prevents parent element from scrolling when hovering over debugger elements.
+ */
+
+function preventDefault(e) {
+    // converts event for all browsers
+    e = e || window.event; 
+    if (e.preventDefault) e.preventDefault(); 
+    e.returnValue = false;
+}
+var scrollElement = function(e,elementID){
```

Comment:
If this was "inspired" from a reference somewhere - then can you add a link to it should we run into any oddities?  If not, don't worry about it.